### PR TITLE
Feature 168 - prepare docker

### PR DESCRIPTION
**Pull request by M-HRL**
#168

### CODE REVIEW

**Comment by garvinkon** [comment]
tutorial: https://github.com/amosproj/amos2023ws01-ticket-chat-ai/wiki/Build---Deploy-Documentation#production

File: backend.log
Comment (irhox) [inline backend.log:1]: I think the log file shouldn't be committed, right?

**Comment by WebFa98** [comment]
Screenshot 2024-01-09 172116
after following the instructions i get the following error when running the project

File: backend.log
Comment (garvinkon) [reply backend.log:1]: yeah, do not know how the backend.log went into the root folder, i put a backend.log into gitignore there aswell.

**Review by WebFa98 — APPROVED** [review]
works for me

**Pull request merged**